packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -91,7 +91,7 @@ export async function bundleServer(buildOpts: BuildOptions): Promise<void> {
       setWranglerExternal(),
       fixRequire(updater),
       handleOptionalDependencies(optionalDependencies),
-      patchLoadInstrumentation(updater),
+      ...patchInstrumentation(updater, buildOpts),
```

Comment:
With my suggestion in the patch file
```suggestion
      patchInstrumentation(updater, buildOpts),
```